### PR TITLE
feat: Configure layout

### PR DESCRIPTION
**Pull request by tajespasarela**
EX-6359

### CODE REVIEW

**Comment by ajperezbau** [comment]
At certain window sizes the env selector overlaps the related tags arrows.

https://user-images.githubusercontent.com/75546736/174794832-f725252a-abdf-41f5-aa81-4a8dce6319a5.mov

**Review by ajperezbau — CHANGES_REQUESTED** [review]

**Comment by tajespasarela** [comment]
> At certain window sizes the env selector overlaps the related tags arrows.
> 
>  Screen.Recording.2022-06-21.at.13.02.24.mov

Not part of this task

File: src/components/desktop/desktop.vue [hunk 1]
```diff
@@ -1,68 +1,63 @@
...
+        >
+          <SearchBox />
 
-        <LocationProvider location="predictive_layer">
-          <RelatedTags v-if="!$x.isEmpathizeOpen && $x.relatedTags.length > 0" />
-        </LocationProvider>
-      </div>
-    </template>
+          <LocationProvider location="predictive_layer">
+            <PredictiveLayer />
```
[the remark this answers is not in the document]
Comment (tajespasarela) [reply src/components/desktop/desktop.vue:15]: fixed

File: package.json [hunk 2]
```diff
@@ -65,6 +65,10 @@
     "http-server": "^14.1.0",
     "lint-staged": "~11.2.6",
     "postcss": "~8.3.11",
+    "postcss-dir-pseudo-class": "^6.0.4",
```
Comment (diegopf) [inline package.json:68]: We should use install `postcss-dir-pseudo-class` with the `~6.0.4` patch releases modifier like the other ones, right?

File: build/instrumentation.build.js [hunk 3]
```diff
@@ -114,8 +99,12 @@ export function createConfig({
...
       styles(
         mergeConfig('styles', {
-          mode: extractCss ? ['extract', outputCss] : 'inject',
-          plugins: postCSSPlugins
+          mode: [
+            'inject',
+            (varname, id) =>
+              // eslint-disable-next-line max-len
+              `import {createInjector} from 'vue-runtime-helpers';const injector=createInjector({});injector('${id}',{source:${varname}})`
+          ]
```
Comment (diegopf) [inline build/instrumentation.build.js:107]: I'm just curious, why do we need this now?

**Review by ajperezbau — APPROVED** [review]

File: src/components/search/facets/facets.vue [hunk 4]
```diff
@@ -1,75 +1,77 @@
 <template>
   <Facets>
```
[the remark this answers is not in the document]
Comment (tajespasarela) [reply src/components/search/facets/facets.vue:2]: to the facets task

File: package.json (hunk 2 above)
Comment (tajespasarela) [reply package.json:68]: fixed

File: build/instrumentation.build.js (hunk 3 above)
Comment (tajespasarela) [reply build/instrumentation.build.js:107]: to use the same way of injecting the CSS as the Vue components

Comment (tajespasarela) [reply build/instrumentation.build.js:107]: necessary to avoid issues, also for future Tailwindcss use

**Review by ajperezbau — APPROVED** [review]

**Review by diegopf — APPROVED** [review]

**Pull request merged**